models/QtiRunnerInitDataBuilder.php

```diff
@@ -149,13 +167,25 @@ protected function getTestMap(QtiRunnerServiceContext $context)
                     $itemData = $this->qtiRunnerService->getItemData($context, $item->getHref());
 
                     $itemId = $item->getIdentifier();
+                    $score = $itemsStates[$itemId]['score'] ?? [];
+                    $state = $itemsStates[$itemId]['state'] ?? [];
+
+                    $answers = 0;
+                    if (count($state)) {
+                        foreach ($state as $response) {
+                            $answers += count($response['response']['list']['identifier'] ?? []);
+                        }
+                    }
+
                     $items[$itemId] = [
                         'id' => $itemId,
                         'label' => $itemData['data']['attributes']['label'],
                         'position' => $position,
                         'categories' => [],
-                        'score' => 0,
-                        'maxScore' => 0
+                        'informational' => empty($state) && empty($score),
+                        'skipped' => empty($answers),
+                        'score' => array_sum($score),
+                        'maxScore' => count($score)
```

Comment:
Still not the proper way to get the max score.
Please target the variable MAX_SCORE from the results.